spread.yaml

```diff
@@ -11,11 +11,43 @@ environment:
     GOVERSION: 1.10.4
     GOHOME: /home/test
     GOPATH: $GOHOME
+    PROJECT_PATH: $GOHOME/src/github.com/snapcore/spread
     PATH: $GOHOME/bin:$PATH
     CACHE_DISABLED: $(HOST:[ "$SEND_CACHE" = 1 ] && echo false || echo tests/cache)
 
 backends:
     google:
+        key: "$(HOST: echo $SPREAD_GOOGLE_AUTH_JSON)"
+        location: computeengine/us-east1-b
+        halt-timeout: 2h
+        systems:
+            - ubuntu-14.04-64:
```

Comment:
I would probably trim this list to include: ubuntu-latest, fedora-latest, opensuse-latest and maybe debian. The rest seem like waste as we're not actually using them for development.